actiontext/test/dummy/db/migrate/2018052816_create_action_text_tables.rb

```diff
@@ -2,7 +2,7 @@ class CreateActionTextTables < ActiveRecord::Migration[6.0]
   def change
     create_table :action_text_rich_texts do |t|
       t.string     :name, null: false
-      t.text       :body, limit: 16777215
+      t.text       :body, size: :medium
```

Comment:
Need to change this in [actiontext/db/migrate/201805281641_create_action_text_tables.rb](https://github.com/rails/rails/blob/master/actiontext/db/migrate/201805281641_create_action_text_tables.rb) as well?

Reply:
ah... you are right!